iogt/views.py

```diff
@@ -140,10 +141,10 @@ def get(self, request, page_id):
             {'name': 'icons', 'extensions': ('.svg',)},
         ]
         for static_dir in static_dirs:
-            for root, dirs, files in os.walk(Path(settings.STATIC_ROOT) / static_dir['name']):
+            for root, dirs, files in os.walk(static_root_path / static_dir['name']):
                 for file in files:
                     if file.endswith(static_dir['extensions']):
-                        static_urls.append(f'{settings.STATIC_URL}{root.split("/static/")[-1]}/{file}')
+                        static_urls.append(f'{settings.STATIC_URL}{root.split(f"{static_root_path.name}/")[-1]}/{file}')
```

Comment:
...and use pathlib's [relative_to method](https://docs.python.org/3.8/library/pathlib.html#pathlib.PurePath.relative_to)?

I think it would make the code much easier to read.

Reply:
refactored the static assets URL generation accordingly